contrib/aws/aws-sdk-go-v2/aws/example_test.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package aws
+
+import (
+	"context"
+	"log"
+
+	awscfg "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+)
```

Comment:
Let's change the package name and import the contrib package like we do in the other contrib examples.

https://github.com/DataDog/dd-trace-go/blob/v1/contrib/aws/aws-sdk-go/aws/example_test.go#L6-L12